dockstore-webservice/src/main/java/io/dockstore/webservice/Helper.java

```diff
@@ -16,15 +16,276 @@
  */
 package io.dockstore.webservice;
 
+import com.esotericsoftware.yamlbeans.YamlReader;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.base.Optional;
+import com.google.gson.Gson;
+import io.dockstore.webservice.core.Container;
+import io.dockstore.webservice.core.Tag;
+import io.dockstore.webservice.core.Token;
+import io.dockstore.webservice.core.TokenType;
 import io.dockstore.webservice.core.User;
+import io.dockstore.webservice.jdbi.ContainerDAO;
+import io.dockstore.webservice.jdbi.TagDAO;
+import io.dockstore.webservice.jdbi.TokenDAO;
+import io.dockstore.webservice.jdbi.UserDAO;
+import io.dockstore.webservice.resources.ResourceUtilities;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 import javax.ws.rs.WebApplicationException;
 import org.apache.http.HttpStatus;
+import org.apache.http.client.HttpClient;
+import org.eclipse.egit.github.core.Repository;
+import org.eclipse.egit.github.core.RepositoryContents;
+import org.eclipse.egit.github.core.client.GitHubClient;
+import org.eclipse.egit.github.core.service.ContentsService;
+import org.eclipse.egit.github.core.service.OrganizationService;
+import org.eclipse.egit.github.core.service.RepositoryService;
+import org.eclipse.egit.github.core.service.UserService;
+import org.slf4j.Logger;
 
 /**
  *
  * @author xliu
  */
 public class Helper {
+    public static class RepoList {
+
+        private List<Container> repositories;
+
+        public void setRepositories(List<Container> repositories) {
+            this.repositories = repositories;
+        }
+
+        public List<Container> getRepositories() {
+            return this.repositories;
+        }
+    }
+
+    public static class FileResponse {
+        private String content;
+
+        public void setContent(String content) {
+            this.content = content;
+        }
+
+        public String getContent() {
+            return this.content;
+        }
+    }
+
+    @SuppressWarnings({ "checkstyle:methodlength", "checkstyle:parameternumber" })
```

Comment:
Break this up rather than just suppress the warning. 
This method is indeed too long and hard to understand. 
There are at least three things this method seems to be doing
1) communicating with quay.io
2) looking through the results for updated containers
3) storing the updated tags

But I could be wrong! Break this up into at least three methods that are documented via Javadoc so that we understand what is going on here.